app/views/admin/categories/index.html.erb

```diff
@@ -25,8 +25,8 @@
           				<td><%= category.title %></td>
           				<td><%= category.description %></td>
           				<td>0</td>
-          				<td class="a"><%= link_to "EDIT", "#", class: 'edit' %></td>
-          				<td class="a"><%= link_to "REMOVE", "#", class: 'remove' %></td>
+          				<td class="a"><%= link_to "EDIT", "categories/#{category.id}/edit", class: 'edit' %></td>
+          				<td class="a"><%= link_to "REMOVE", "categories/#{category.id}/destroy" ,{method: "delete"}, class: 'remove' %></td>
```

Comment:
Please remove the `{}` brackets in `{method: delete}`.
Then change `"categories/#{category.id}/destroy"` to `admin_category_path(category)`